.github/actions/install-python-packages/action.yml

```diff
@@ -0,0 +1,23 @@
+name: 'install-python-packages'
+description: 'Install necessary python packages'
+runs:
+  using: "composite"
+  steps:
+    - id: install-python-packages
+      run: |
+        sudo pip3 install -U \
+          sphinx==3.0.3 \
+          doc8==0.8.0 \
+          sphinx_rtd_theme==0.4.3 \
+          sphinxcontrib.spelling==5.0.0 \
+          sphinxcontrib-imagehelper==1.1.1 \
+          sphinx-tabs==3.2.0 \
+          colcon-common-extensions \
+          colcon-mixin \
+          vcstool \
+          GitPython \
+          setuptools \
+          gcovr==5.0 \
+          pyyaml \
+          jsonschema
```

Comment:
NIT: We might want to reduce the dependencies to what it's actually required.

Reply:
TODO